ttnn/cpp/ttnn/operations/eltwise/unary/common/unary_op_utils.cpp

```diff
@@ -922,7 +922,7 @@ std::map<std::string, std::string> get_block_defines(
 
 // update split eltwise ops include macros
 void update_macro_defines(UnaryOpType op_type, std::map<std::string, std::string>& defines) {
-    defines[get_macro_definition(op_type)] = "1";
+    defines[get_macro_definition(op_type)] = std::string("1");
```

Comment:
This is just silly on GCC's part. It's statically provable that this won't overflow. If it weren't then it would be warning about this constructor call too.
Maybe something that feels a little less like unnecessary boilerplate and more like conventional C++ might be:
```cpp
using namespace std::string_literals;
defines[get_macro_definition(op_type)] = "1"s;
```
wdyt?